documentation/modules/upgrading/proc-upgrade-kafka-kraft.adoc

```diff
@@ -0,0 +1,130 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-upgrade-kafka-kraft-{context}']
+= Upgrading KRaft-based Kafka clusters and client applications
+
+[role="_abstract"]
+Upgrade a KRaft-based Strimzi Kafka cluster to the latest supported Kafka version and KRaft metadata version.
```

Comment:
As people might not always upgrade only to the latest version, maybe this is clearer?

```suggestion
Upgrade a KRaft-based Strimzi Kafka cluster to a newer supported Kafka version and KRaft metadata version.
```